README.md

```diff
@@ -41,7 +41,7 @@ var app = express();
 // setup your routes
 // `options` are set to default values. For more info, see `options` below.
 // app.use(errorHandler({ /* options, see below */ }));
-app.use(errorHandler({debug:true, log:true}));
+app.use(errorHandler({debug: app.get('env') === 'development', log:true}));
```

Comment:
I think it would be better to split this onto multiple lines, put `debug` and `log` on their own lines.